exporter_app/tasks/export_ead_task.rb

```diff
@@ -46,11 +58,36 @@ def load_into_work_queue(updates)
     #
     # -- Mark (Wednesday 20 April  15:38:14 AEST 2016)
     #
+    # Still don't seem to need it ...
+    #
+    # -- James (Fri Apr 29 14:31:38 AEST 2016)
+    #
     updates['removes'].each do |remove_id|
       @work_queue.push('remove', remove_id)
     end
 
     @work_queue.optimize
   end
 
+  def ead_export_directory
+    exp_dir = File.join(@workspace_directory, EXPORTED_DIR)
+    Dir.mkdir(exp_dir) unless Dir.exist?(exp_dir)
```

Comment:
Minor thing, but worth switching to:

  FileUtils.mkdir_p(exp_dir)

here.  It's not an issue in this case, but the general pattern of checking existence before calling mkdir can create subtle race conditions when there are multiple processes.  Might as well use the safer mkdir_p...